cmd/podRestartsExternalCheck/podRestartsExternalCheck.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: pod-restarts-check
+  namespace: kuberhealthy
+spec:
+  runInterval: 60m
+  timeout: 58m
+  podSpec:
+    containers:
+      - env:
+          - name: POD_NAMESPACE
+            value: "kube-system"
+          - name: CHECK_RUN_WINDOW
+            value: "55m"
+        image: docker-proto.repo.theplatform.com/pod-restarts-check:1.0.0
```

Comment:
need to change image location

Reply:
I have invited you to the quay.io account.  You can make a new public repository there and push these images in.